docs/content/policy-reference.md

```diff
@@ -997,6 +998,7 @@ The `aws_config` object parameter may contain the following fields:
 | `aws_service` | yes | `string` | AWS service the request will be valid for. (e.g. `"s3"`) |
 | `aws_region` | yes | `string` | AWS region for the request. (e.g. `"us-east-1"`) |
 | `aws_session_token` | no | `string` | AWS security token. Used for the `x-amz-security-token` request header. |
+| `disable_payload_signing` | no | `boolean` | When `true` an `UNSIGNED-PAYLOAD` value will be used for calculating the `x-amz-content-sha256` header during signing, and will be returned in the response. Default: `false`. |
```

Comment:
For consistency should we prefix this option with `aws` like the others?

Reply:
since it is already under aws_config, not sure if aws_ prefix is required. This config is also more specific to s3 and not generic like other configs.

Reply:
> This config is also more specific to s3 and not generic like other configs.

If that's the case, we should at least update the description to clarify that.